algorithms/src/snark/marlin/ahp/prover/round_functions/first.rs

```diff
@@ -94,7 +95,7 @@ impl<F: PrimeField, MM: MarlinMode> AHPForR1CS<F, MM> {
             let c_domain = circuit_state.constraint_domain;
             let i_domain = circuit_state.input_domain;
 
-            let mut circuit_r_b_s = Vec::with_capacity(batch_size);
+            let mut circuit_r_b_s = Vec::with_capacity(z_a.len());
```

Comment:
Why did this change? We do still have `batch_size` in scope

Reply:
5ebe7b258

Reply:
5ebe7b258